Changelogs for Digital Twin Manager Plugins 5.0.0 (#4136)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1024-0-0-fix-device-property-type-and-date-handling.md

---
date: ""

> title: "Fix device property type and date handling."

Reviewer: ```suggestion
title: "Fixed device property data types and date parsing in Asset Properties widget"
```

> title: "Fixed device property data types and date parsing in Asset Properties widget"

Reviewer 2: ```suggestion
title: "Fixed device property data type and date parsing in Asset properties widget"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2671"
version: "1024.0.0"
---

> Previously, device properties in the Asset Properties widget were stored

Reviewer 2: ```suggestion
Previously, device properties in the "Asset properties" widget were stored
```

with an incorrect type, and date fields in complex properties displayed

> an Invalid Date value during editing; now, property data is stored using

Reviewer 2: ```suggestion
an invalid date value during editing. Now, property data is stored using
```
